template-parts/content-video.php

```diff
@@ -21,7 +21,10 @@
 
 		<?php the_title( sprintf( '<h2 class="entry-title"><a href="%s" rel="bookmark">', esc_url( get_permalink() ) ), '</a></h2>' ); ?>
 
-		<?php echo esc_html( get_the_content( '', false ) ); ?>
```

Comment:
Need to adjust this, as by default if there is no excerpt it should take the content as an excerpt.

Reply:
That's exactly why the original issue has happened. In conjunction with `esc_html` it outputs the HTML markup. See
http://i.imgur.com/BvBkPJk.png
http://i.imgur.com/eiG5LvW.png

Reply:
Then maybe lets use strip_tags ? 
The idea is that if users didn't add an excerpt, it should be generated from the content.  At the moment t didn't work for me like that. 🤔

Reply:
Looks like `the_excerpt` works more intelligently in this case. It extracts the part of the content if an excerpt is not provided. See https://imgur.com/Ir0L5Tf.png